examples/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 if(CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
   cmake_minimum_required(VERSION 3.12)
   project(ArborXExamples CXX)
-  find_package(ArborX 1.1 REQUIRED)
+  find_package(ArborX 1.2 REQUIRED)
```

Comment:
Why are we passing a version number in the first place?

Reply:
https://github.com/arborx/ArborX/pull/521

Reply:
I don't see a harm in this. In this way, we at least make sure that no old version is picked up when trying to build the bundled examples.